solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java

```diff
@@ -2871,10 +2871,30 @@ public static LBHttpSolrClient getLBHttpSolrClient(String... solrUrls)
     return new LBHttpSolrClient.Builder().withBaseSolrUrls(solrUrls).build();
   }
 
+  /** This method creates a HttpClient from a URL. */
+  public static HttpClient getHttpClient(String url) {
+    return new HttpSolrClient.Builder(url).build().getHttpClient();
+  }
+
   /**
-   * This method <i>may</i> randomize unspecified aspects of the resulting SolrClient. Tests that do
-   * not wish to have any randomized behavior should use the {@link
-   * org.apache.solr.client.solrj.impl.HttpSolrClient.Builder} class directly
+   * This method creates a basic HttpSolrClient. Tests that want to control the creation process
+   * should use the {@link org.apache.solr.client.solrj.impl.HttpSolrClient.Builder} class directly
+   */
```

Comment:
makes sense.   adding language..  I don't have a recommendation of what to move too since we aren't I think quite there yet...  future PR move everyone to the jetty httpclient?

Reply:
HttpSolrClient and CloudLegacySolrClient are already deprecated and many methods that return them are likewise.  It's not clear to me why tests can't use the Http2 (Jetty) one; can you tell me?

Reply:
i actually did just that, and then took all of those changes out based on review feedback.    Part of why I want to get this merged is cause I can't remember all the twists and turns we've made, and so I'ld love to get this in, and then in the next one revisit moving to Http2SolrClient etc...